src/decaylanguage/dec/dec.py

```diff
@@ -1813,3 +1818,13 @@ def _str_to_bool(arg: str) -> bool:
     raise ValueError(
         f"String {arg!r} cannot be converted to boolean! Only 'yes/no' accepted."
     )
+
+
+def _edit_model_name_terminals(t: TerminalDef) -> None:
+    """
+    Edits the terminals of the grammar to replace the model name placeholder with the actual names of the models.
+    """
+
+    modelstr = rf"(?:{'|'.join(re.escape(kdm) for kdm in sorted(known_decay_models, key=len, reverse=True))})"
```

Comment:
I'm sorting the models since that's what Lark does. It also works without sorting but then a bunch of tests fail which seem to rely on this sorting.

Reply:
There is some griddy pattern matching, I recall, which explained the sorting. For example, the parking might find XXX in model XXXYYY even if a model XXX also exists, and then the latter would not be parsed correctly. I may be mistaken but ...